### DISCUSSION TITLE
This package works with Laravel Sail?

### DISCUSSION THREAD

**Discussion by icarojobs** [worksome/envy#9] [Q&A]
Apparently, this package doesn't work with Laravel Sail (or I don't know how to configure it).

I'm testing here and it's not working.

**Comment by olivernybroe** [comment]
Hi @icarojobs, could you add some more information on what you mean it doesn't work? What error message are you getting and how are you trying to run it with sail?

**Comment by usernotnull** [comment]
Yes it does, I'm on sail and working fine @icarojobs
